README.rst

```diff
@@ -3,469 +3,66 @@ Watson
 
 |Build Status| |PyPI Downloads Per Month| |PyPI Latest Version| |Requires.io|
 
-Watson is here to help you monitoring your time. You want to know how
+Watson is here to help you manage your time. You want to know how
 much time you are spending on your projects? You want to generate a nice
 report for your client? Watson is here for you.
 
-Tell Watson when you start working on a task with the ``start`` command.
-Then, when you are done with this task, stop the timer with the ``stop``
-command. This will create what we call a **frame**. That's pretty much
-everything you need to know to start using Watson.
+Wanna know what it looks like? Check this below.
 
-Each frame consists of the name of a project and some tags. Your tags
-can be shared across projects and can be used to generate detailed
-reports.
+![screenshot](docs/img/screenshot.png)
```

Comment:
Finally it wasn't :)
https://github.com/TailorDev/Watson/commit/d168b2b79f3bc521fd4ed7198cb26e0a4e42e973